Sources/Steem/API.swift

```diff
@@ -60,6 +60,18 @@ public struct API {
         public let method = "get_dynamic_global_properties"
         public init() {}
     }
+    
+    public struct SteemPrices: Decodable {
+        public let steemSbd: Int32
+        public let steemUsd: Int32
+        public let steemVest: Int32
```

Comment:
Prices are floating point

Reply:
Ok Cool, and is this the right track for getting the new conveyor endpoints into the Wallet?

Reply:
Yeah for now at least, we’ll probably lift non steemd calls to the shared lib later

Reply:
NVM, I see it's already hooked up there.
https://github.com/steemit/swift-steem/blob/b6e714c0054d0e0bd91c84073f50023c761647c7/Tests/SteemIntegrationTests/API.swift#L4-L8